gradle.properties

```diff
@@ -1,7 +1,10 @@
-grailsVersion=5.2.1
-gorm.version=7.3.2
+grailsVersion=6.0.0
+grailsGradlePluginVersion=6.0.0
+gormVersion=8.0.0
```

Comment:
hibernate5 dependency that is imported does not match the new gormVersion. Was that tested? This differs from documentation https://gorm.grails.org/latest/hibernate/manual/